charmhelpers/contrib/network/ovs/__init__.py

```diff
@@ -343,6 +345,14 @@ def add_ovsbridge_linuxbridge(name, bridge, ifdata=None):
                20dac08fdcce4b7fda1d07add3b346aa9751cfbc/
                    lib/db-ctl-base.c#L189-L215
     :type ifdata: Optional[Dict[str,Union[str,Dict[str,str]]]]
+    :param portdata: Additional data to attach to port. Similar to ifdata.
+    :type portdata: Optional[Dict[str,Union[str,Dict[str,str]]]]
+
+    WARNING: the `ifup` command (NetworkManager) must be available on the system
```

Comment:
Shouldn't we detect that it's not available and use the alternative on alter Ubuntu releases?